data/mods/Aftershock/mobs/monster_faction.json

```
@@ -2,56 +2,80 @@
   {
     "type": "MONSTER_FACTION",
     "name": "PrepNet",
+    "base_faction": "human",
     "friendly": [ "Prepnet_Phyle", "PrepNet" ],
-    "neutral": [ "herbivore", "human", "player" ],
-    "by_mood": [ "insect", "small_animal" ],
+    "neutral": [ "herbivore", "human" ],
+    "by_mood": [ "insect", "small_animal", "WraitheonRobotics" ],
     "hate": [ "fungus" ]
   },
   {
     "type": "MONSTER_FACTION",
     "name": "WraitheonRobotics",
-    "neutral": [ "cop_bot", "defense_bot", "utility_bot", "animal", "uplift", "robofac" ],
-    "hate": [ "zombie", "fungus", "cult", "triffid", "nether", "PrepNEt", "moxie", "bio_machine" ]
+    "base_faction": "robofac",
+    "neutral": [ "cop_bot", "defense_bot", "utility_bot", "animal", "UPLIFT", "robofac" ],
+    "by_mood": "human",
+    "hate": [ "zombie", "fungus", "cult", "triffid", "nether", "PrepNet", "moxie", "bio_machine" ]
   },
   {
     "type": "MONSTER_FACTION",
     "name": "whately",
-    "friendly": [ "human", "player", "whately", "whately_family" ],
+    "base_faction": "human",
+    "friendly": [ "human", "whately", "whately_family" ],
     "neutral": "herbivore",
     "by_mood": [ "insect", "small_animal" ],
     "hate": "zombie"
   },
   {
     "type": "MONSTER_FACTION",
     "name": "UPLIFT",
-    "neutral": "herbivore",
-    "by_mood": [ "insect", "small_animal", "player" ],
+    "base_faction": "bear",
+    "neutral": [ "herbivore", "robofac" ],
+    "by_mood": [ "insect", "small_animal", "human" ],
     "hate": "zombie"
   },
   {
     "type": "MONSTER_FACTION",
     "name": "moxie",
+    "base_faction": "mutant",
     "neutral": "bio_machine",
-    "hate": [ "small_animal", "herbivore", "insect", "player" ]
+    "hate": [ "small_animal", "herbivore", "insect", "human", "mutant", "WraitheonRobotics", "bot" ]
```

Suggestion re moxies:
```suggestion
    "by_mood": [ "small_animal", "herbivore" ],
    "hate": [ "insect", "human", "mutant", "WraitheonRobotics", "bot" ]
```
Given their high aggression, they'll still be attacking small animals and herbivores - humans, cryo, etc will just be priorities over those. A further change would just put "animal" in "by_mood".